src/testUtils/mocks/dataSources.ts

```diff
@@ -584,7 +584,17 @@ const defaultContextOptions: ContextOptions = {
     next: 1,
     count: 1,
   },
-  primaryKey: 'primaryKey',
+  primaryKey: {
+    account: ({
+      address: 'primaryKey',
+    } as unknown) as Account,
```

Comment:
I think it would be nice to not have this type assertion here.

Not sure how feasible it is though.

Reply:
Such type assertions are used at other places as well, hence I used the approach here. It works and doesn't impact anything so seems OK to me.

Reply:
True, OK to leave.